specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -617,7 +614,7 @@ if libraries need to access classes packaged in the application modules.
 Components in the web container must have
 access to the following classes
 
-and resources. Note that as of Java EE 6, EJB
+and resources. Note that as of Jakarta EE 6, EJB
```

Comment:
Maybe this should say Java EE 6 as it is an older version being referred to. Although EJB needs to change

Reply:
reverted.